scripts/migrate-framer-motion.js

```
@@ -0,0 +1,81 @@
+        // Keep track of what's being imported
+        const items = importedItems.split(',').map(item => item.trim());
+        
+        // Create a new import statement using our motion utility
+        return `import { ${items.join(', ')} } from "@/lib/motion";`;
+      }
+    );
+    
+    // Replace type annotations for VariantLabels and TargetAndTransition
+    content = content.replace(
+      /VariantLabels\s*\|\s*TargetAndTransition/g,
+      'Record<string, any>'
```

ESLint

[error] Strings must use doublequote.

(quotes)